cmd/promtool/backfill_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/prometheus/prometheus/tsdb"
+	"github.com/prometheus/prometheus/tsdb/importer/openmetrics"
+	"github.com/prometheus/prometheus/tsdb/tsdbutil"
+	"github.com/prometheus/prometheus/util/testutil"
+)
+
+type sample1 struct {
```

Comment:
`sample1` is not a nice name. Does `sample` collide with anything?

Reply:
It interferes with this https://github.com/prometheus/prometheus/blob/master/cmd/promtool/unittest.go#L500 (The test compiles and runs ok if I rename sample1 -> sample but my editor shows this message ```sample redeclared in this block previous declaration at ./unittest.go:514:6)```.

Reply:
I could name it as sample if we want to ignore the editor's message or I could rename it to backfillSample ?